modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_variables.scss

```diff
@@ -125,25 +125,25 @@ $toggle-card-on-icon-border-width: 0 !default;
 
 // ---------- File icons colors ----------
 
-$file-icon-color-0: $white !default;
-$file-icon-color-1: $white !default;
-$file-icon-color-2: $white !default;
-$file-icon-color-3: $white !default;
-$file-icon-color-4: $white !default;
-$file-icon-color-5: $white !default;
-$file-icon-color-6: $white !default;
-$file-icon-color-7: $white !default;
+$file-icon-color-0: $gray-600 !default;
+$file-icon-color-1: $gray-600 !default;
+$file-icon-color-2: $teal-l1 !default;
+$file-icon-color-3: $red-l1 !default;
+$file-icon-color-4: $orange-l1 !default;
+$file-icon-color-5: $purple-l1 !default;
+$file-icon-color-6: $cyan-l1 !default;
+$file-icon-color-7: $pink-l1 !default;
 
 // ---------- File icons text colors ----------
 
-$file-icon-text-color-0: #6b6c7e !default;
-$file-icon-text-color-1: #a7a9bc !default;
-$file-icon-text-color-2: #50d2a0 !default;
-$file-icon-text-color-3: #af78ff !default;
-$file-icon-text-color-4: #ffb46e !default;
-$file-icon-text-color-5: #ff5f5f !default;
+$file-icon-text-color-0: $gray-600 !default;
+$file-icon-text-color-1: $gray-500 !default;
+$file-icon-text-color-2: $teal !default;
+$file-icon-text-color-3: $purple !default;
+$file-icon-text-color-4: $orange !default;
+$file-icon-text-color-5: $red !default;
 $file-icon-text-color-6: #4b9bff !default;
```

Comment:
@pat270 why is this the only color not being updated to a sass variable?

Reply:
There was no match for that particular color in the spread sheet https://docs.google.com/spreadsheets/d/1SF0_nFyXVPbZCFrOyDt5sTe0vg-rt8it9AhdHKtTbac/edit?pli=1&gid=591426718#gid=591426718. $blue-l1 is similar.

Reply:
Hey, correct me if I am wrong, but the colors are:

![image](https://github.com/user-attachments/assets/9dee0d81-af82-442f-9bcc-45440dfc9816)

Reply:
@pat270 can you test this and share a screenshot of the final result. You should be able to reproduce this uploading files of the types above to the document library. Thanks.

Reply:
I updated the pr. I changed the wrong variables and made the background's all white. There is a bug where the correct `file-icon-color-#` class isn't being implemented. I had to manually change the classes for this screenshot.

![file-icon-color](https://github.com/user-attachments/assets/780724b4-e81e-46ba-92e7-1959f035d355)

Reply:
@pat270 what's the bug? Can you create a ticket for it?